src/AppBundle/Controller/Admin/UserController.php

```diff
@@ -38,8 +38,9 @@ public function viewAction(Request $request, User $user)
             new AdminUserType(),
             $user,
             [
-                'manager' => $this->getDoctrine()->getManager(),
-                'locale' => $request->getLocale(),
+                'manager'       => $this->getDoctrine()->getManager(),
+                'locale'        => $request->getLocale(),
+                'wantToLearn'   => (int)$user->getWantToLearn()
```

Comment:
@jongotlin Passing false in data property does not pre-select the radio button. You are already over reacting on your code review. You should be focusing to make the code readable. Those functions are added in PHP be used. Can I see your code?